tests/e2e/orchestrator/orchestrator.go

```diff
@@ -41,7 +45,7 @@ const (
 	ojoGrpcPort       = "9090"
 	ojoMaxStartupTime = 40 // seconds
 
-	priceFeederContainerRepo  = "ghcr.io/ojo-network/price-feeder-ojo"
+	priceFeederContainerRepo  = "ghcr.io/ojo-network/price-feeder-ojo-47"
```

Comment:
I made a GH issue to change this back: https://github.com/ojo-network/ojo/issues/238

Let's add a TODO here ❤️